docs/docs/cli/running-tests.md

```diff
@@ -1,23 +1,30 @@
 # Running Tests From the Command Line Interface (CLI)
-Once you have created a test, whether from the Tracetest UI or via a text editor, you will need the capabity to run it via the Command Line Interface (CLI) to integrate it into your CI/CD process or your local development workflow. The documentation for running a test via the CLI can be found here: [tracetest test run](./reference/tracetest_test_run.md). This page will provide some examples of using this command.
+
+Once you have created a test, whether from the Tracetest UI or via a text editor, you will need the capabity to run it via the Command Line Interface (CLI) to integrate it into your CI/CD process or your local development workflow.
+
+The documentation for running a test via the CLI can be found here:
+
+- [tracetest test run](./reference/tracetest_test_run.md): This page provides examples of using this command.
 
 ## Running Your First Test
-To run a test, give the path to the test definition file with the '-d' option. This will launch a test, providing us with a link to the created test run.
 
+To run a test, give the path to the test definition file with the `'-d'` option. This will launch a test, providing us with a link to the created test run.
```

Comment:
```suggestion
To run a test, give the path to the test definition file with the `'-d'` option. This will launch a test and provide with link to the created test run.
```